### PR TITLE
Add missing network methods to ain-js

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add getChainId() method
- Add getConsensusStatus() method 
- Add getRawConsensusStatus() metod
- Add getPeerCandidateInfo() method 
- Fix test endpoint port number 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1248

### CODE REVIEW

**Review by jiyoung-an — APPROVED** [review]
LGTM with some questions, thanks!

File: src/net.ts [hunk 1]
```diff
@@ -21,32 +21,33 @@ export default class Network {
...
   }
 
   /**
-   * Fetches the protocol version of the blockchain node.
+   * Checks whether the blockchain node is syncing with the network or not.
+   * @returns {Promise<boolean>}
    */
-  getProtocolVersion(): Promise<any> {
-    return this.provider.send('ain_getProtocolVersion');
+  isSyncing(): Promise<boolean> {
```
[the remark this answers is not in the document]
Comment (platfowner) [reply src/net.ts:49]: 'connected' blockchain node means it has at least 1 peer node, and 'syncing' blockchain node means it's try to catch up the block-chain by fetching missing blocks from peer nodes.

[the remark this answers is not in the document]
Comment (jiyoung-an) [reply src/net.ts:49]: Oh i see! thank you for explain!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an  Now merging..

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Pull request merged**